jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SubnetInventoryCollector.java

```diff
@@ -54,6 +53,10 @@ public List<SubnetVH> fetchSubnetDetails(SubscriptionVH subscription) {
 					JsonObject subnetObject = subnetElement.getAsJsonObject();
 					JsonObject properties = subnetObject.getAsJsonObject("properties");
 					subnetVH.setId(subnetObject.get("id").getAsString());
+					String id =subnetVH.getId();
+					int beginningIndex=subnetVH.getId().indexOf("resourceGroups")+15;
+					String resourceGroupName=(subnetVH.getId()).substring(beginningIndex,id.indexOf('/',beginningIndex+2));
+					subnetVH.setResourceGroupName(resourceGroupName);
```

Comment:
Can we refactor this to a utility method

Reply:
Moved to util class